blobstore/blobstore.go

```diff
@@ -106,23 +111,67 @@ func handler(r *data.FileRepo) http.Handler {
 func main() {
 	defer shutdown.Exit()
 
+	usage := `
+usage: flynn-blobstore <command> [<args>...]
+
+Commands:
+        help        show usage for a specific command
+        cleanup     delete file blobs from default backend 
+        migrate     move file blobs from default backend to a different backend
+        server      run blobstore HTTP server
+
+See 'flynn-blobstore help <command>' for more information on a specific command.
+`[1:]
+	args, _ := docopt.Parse(usage, nil, true, version.String(), true)
+
+	cmd := args.String["<command>"]
+	cmdArgs := args.All["<args>"].([]string)
+
+	if cmd == "help" {
+		if len(cmdArgs) == 0 { // `flynn-blobstore help`
+			fmt.Println(usage)
+			return
+		} else { // `flynn-blobstore help <command>`
+			cmd = cmdArgs[0]
+			cmdArgs = make([]string, 1)
+			cmdArgs[0] = "--help"
+		}
+	}
+
+	if cmd == "server" {
```

Comment:
You should register `server` like the other two commands so that `flynn-blobstore help server` works.